integration/testnet/network.go

```diff
@@ -596,6 +596,11 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig) *FlowNetwork {
 	}
 	require.GreaterOrEqualf(t, len(accessNodeIDS), DefaultMinimumNumOfAccessNodeIDS,
 		fmt.Sprintf("at least %d access nodes that are not a ghost must be configured for test suite", DefaultMinimumNumOfAccessNodeIDS))
+	for _, n := range confs {
+		if n.Role == flow.RoleObserverService && !n.Ghost {
+			accessNodeIDS = append(accessNodeIDS, n.NodeID.String())
+		}
+	}
```

Comment:
I don't think we want to include observers in this list, since it's used by collection and consensus nodes to send transactions

Reply:
I need to run a separate test without this. What I believe happened was that `make init` crashed in the Flow Virtual Machine with observers. I think that was an older version though. I will address this.